public/locale/en.json

```
@@ -1557,6 +1560,7 @@
   "middleware_hostname_sourced_from": "Middleware hostname sourced from {{ source }}",
   "mild": "Mild",
   "min_char_length_error": "Must be at least {{ min_length }} characters",
+  "min_character_validation": "Minimum {{length}} characters allowed",
```

```suggestion
  "min_character_validation": "Minimum {{length}} characters required",
```


can we change to required?
or we can have both min and max in one !?

---

oh my bad 😅 
Let's use required for min since very less chance there would be need for max validation to show